docs/advanced_usage.md

```diff
@@ -585,10 +585,13 @@ services:
 ### Case Sensitivity
 
 The database interface does not provide a method to configure a MySQL
-database to be case sensitive. This would prevent a user from creating a
+database to be case sensitive. A case **in**sensitive database prevents a user from creating a
```

Comment:
```suggestion
database to be case-sensitive. A case-**in**sensitive database prevents a user from creating a
```

Reply:
I thought so but it wasn't in the original so I didn't! :D